tests/conftest.py

```diff
@@ -38,3 +39,22 @@ def gitecho(request):
     aliases['git'] = lambda args: 'Would have run: ' + ' '.join(args) + '\n'
     yield None
     del aliases['git']
+
+
+def rmtree(dirname):
```

Comment:
We didn't have this? Nice catch.

Reply:
I think that there is a rmtree in distutils in the standard library

Reply:
Just checked, same problem with remove_tree() from distutils (it's like calling rmtree with `ignore_errors=True`)

Reply:
Ahh fair.  Sorry I didn't read the whole docstring.